mkt/ratings/views.py

```diff
@@ -137,6 +142,7 @@ def post_save(self, obj, created=False):
             mkt.log(mkt.LOG.EDIT_REVIEW, app, obj)
             log.debug('[Review:%s] Edited by %s' %
                       (obj.pk, self.request.user.id))
+        obj.guess_language()
```

Comment:
Perhaps this should add `lang=guess_language(obj.body) or obj.user.lang` to the params before it saves instead of doing a second save after. It would work well with moving the language guessing code out of `mkt.ratings.models` too.